Commands/InteractionCommands/InsidersInteractions.cs

```diff
@@ -0,0 +1,28 @@
+namespace Cliptok.Commands.InteractionCommands
+{
+    public class InsidersInteractions : ApplicationCommandModule
+    {
+        [SlashCommand("send-insiders-info-buttons", "Sends a message with buttons to get Insider roles for #insiders-info.", false)]
+        [SlashRequireHomeserverPerm(ServerPermLevel.TrialModerator), SlashCommandPermissions(permissions: DiscordPermission.ModerateMembers)]
```

Comment:
Due to the fact it only needs to be used once and can have mildly chaotic consequences if misused, this should be restricted to someone who is either lead mod **or** has bot owner rights. (i.e. myself, you, and timewalker)